tests/testthat/indention_curly_brackets/multi_line_curly_round_only-out.R

```diff
@@ -1,7 +1,7 @@
 a <- function(x) {
   x <- c(1,
     2 + 3,
-    sin(pi))
+    sin(pi)) # FIXME add tidyverse-comliant rule to break after '('
```

Comment:
I think we could. The question is whether that is non-invasive. I guess you only refer th the case where the function call spans over multiple lines, right? I suggest to move that discussion to #118.